### PR TITLE
[Display eco-news: Single View]Localization for the eco-news single view

### PR DESCRIPTION
**Pull request by zendermean**

### CODE REVIEW

File: src/app/component/eco-news/components/eco-news-detail/eco-news-detail.component.html [hunk 1]
```diff
@@ -66,7 +66,7 @@
           {{ newsItem.text }}
         </div>
         <div class="news-source">
-          <p>Source:</p><a href="{{ newsItem.source }}"> {{ newsItem.source }}</a>
+          <p>{{ "eco-news-single-view.news-source" | translate}}</p><a href="{{ newsItem.source }}"> {{ newsItem.source }}</a>
```
Comment (KimAT25) [inline src/app/component/eco-news/components/eco-news-detail/eco-news-detail.component.html:69]: Please move tag <a> to new line and alignment in 1 column with tag <p>

File: src/app/component/eco-news/components/eco-news-detail/eco-news-detail.component.html [hunk 2]
```diff
@@ -10,19 +10,19 @@
                  class="button-arrow-img"
```
Comment (KimAT25) [inline src/app/component/eco-news/components/eco-news-detail/eco-news-detail.component.html:10]: Generally please pull all changes from 'dev' branch and resole all conflicts

**Comment by mplesha** [comment]
Just one: provide some convenient sequence for imports (eg. import of angular modules - at first, third-party library - next, imports yours modules -next)

```js
import { Component, OnInit, Output, EventEmitter, OnDestroy } from '@angular/core';
import { ecoNewsIcons } from 'src/assets/img/icon/econews/profile-icons';
import { EcoNewsService } from 'src/app/component/eco-news/services/eco-news.service';
import { EcoNewsModel } from 'src/app/component/eco-news/models/eco-news-model';
import { ActivatedRoute } from '@angular/router';
import { Subscription } from 'rxjs';
```

**Comment by mplesha** [comment]
Resole conflict
image

**Pull request merged**